### ISSUE TITLE
withUrlEncodedBody sets incorrect Content-type

### ISSUE THREAD

**Issue by LimmaPaulus** [lukewestby/elm-http-builder#26]
Hi!
There is typo in Content-type, last 'd' is missing. It should be "x-www-form-urlencoded"
```
in
    withStringBody
        "application/x-www-form-urlencode"
        trimmedQuestionMark
```

**Comment by TheAlemazing** [comment]
[PR here](https://github.com/lukewestby/elm-http-builder/pull/27)

**Comment by daapp** [comment]
Author, apply PR please!

**Comment by lukewestby** [comment]
published 7.0.1
